domain/schema/model.go

```diff
@@ -58,14 +60,18 @@ func ModelDDL() *schema.Schema {
 		changeLogTriggersForTable("storage_volume", "uuid", tableVolume),
 		changeLogTriggersForTable("storage_volume_attachment", "uuid", tableVolumeAttachment),
 		changeLogTriggersForTable("storage_volume_attachment_plan", "uuid", tableVolumeAttachmentPlan),
+
+		changeLogTriggersForTableOnColumn("secret_metadata", "secret_id", "auto_prune", tableSecretMetadata),
 		changeLogTriggersForTable("secret_rotation", "secret_id", tableSecretRotation),
-		changeLogTriggersForTable("secret_revision", "uuid", tableSecretRevisionObsolete),
+		changeLogTriggersForTableOnColumn("secret_revision_obsolete", "revision_uuid", "obsolete", tableSecretRevisionObsolete),
 		changeLogTriggersForTable("secret_revision_expire", "revision_uuid", tableSecretRevisionExpire),
-		changeLogTriggersForTableOnColumn("secret_metadata", "secret_id", "auto_prune", tableSecretAutoPrune),
-		changeLogTriggersForTable(
-			"subnet", "uuid", tableSubnet),
-		changeLogTriggersForTable(
-			"subnet_association", "associated_subnet_uuid", tableSubnetAssociation),
+		// We only want to watch the new revision INSERT.
+		// To achieve this and have less performance impact on the event stream watcher,
+		// we watch the changes on the immutable revision field.
+		changeLogTriggersForTableOnColumn("secret_revision", "uuid", "revision", tableSecretRevision),
```

Comment:
This is no longer true with the change to the tables to split out a secret_revision_obsolete table